src/ol/control/control.js

```diff
@@ -121,3 +124,18 @@ ol.control.Control.prototype.setMap = function(map) {
     map.render();
   }
 };
+
+
+/**
+ * This function is used to set a target element for the control. This
+ * function will have no effect if it is called after the control has
```

Comment:
That may work but it's also three setMap calls if you add the control to the map and then calls setTarget. This will also add the control element to the map overlay container to finally remove it to add it to the final target. Not so nice I think.